docs/artifacts.md

```diff
@@ -511,7 +511,6 @@ Upon resolution and execution of the `TaskRun`, the `Status` will look something
         "enableKeepPodOnCancel": false,
         "enableParamEnum": false,
         "enableProvenanceInStatus": true,
-        "enableStepActions": true,
```

Comment:
Should this be removed?

Reply:
no, you're right the flag does still exist